qmt/physics_constants.py

```diff
@@ -100,12 +86,17 @@ def to_float(expr):
     return float(cancel(expr))
 
 
+def to_float_vec(arr):
```

Comment:
should we just vectorize to_float?

Reply:
So I tried that, then `to_float(0.1)` would give `array(0.1)`, which isn't exactly what we want?